lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -59,6 +59,10 @@ final class ComicFormViewController: UIViewController, AlertDisplaying {
     private let cameraViewController = CameraViewController()
     private let disposeBag = DisposeBag()
 
+    // MARK: - Variables
+
+    var delegate: DismissViewController?
```

Comment:
Let's rename DismissViewController to ViewControllerDismissingDelegate . DismissViewController sounds like an actual ViewController that might exist in our codebase.